.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Associated Jira ticket: # (issue)
+
+Please include a summary of the change, what this fixes/creates/improves.
+
+
+# How to test the PR
+
+Please include steps to test your PR.
+
+# Before the change
+
+
+# After the change
+
+
+# Checklist:
+
+- [ ] The commit message has the Jira ticket linked
+- [ ] PR has a short description
+- [ ] Screenshots before and after the change are added
+- [ ] Tests for the changes have been added
+- [ ] Read.me is updated if necessary
```

Comment:
Maybe as an optional point in the checklist/testing requirements, we could add eventual dependencies. For example when something needs a PR from frontend-components or it needs to be run with another app together.

Reply:
@bastilian it makes sense. What would be the best wording for this?